dbms/tests/queries/0_stateless/00502_sum_map.sql

```diff
@@ -7,6 +7,7 @@ INSERT INTO sum_map VALUES ('2000-01-01', '2000-01-01 00:00:00', [1, 2, 3], [10,
 
 SELECT * FROM sum_map ORDER BY timeslot;
 SELECT sumMap(statusMap.status, statusMap.requests) FROM sum_map;
+SELECT sumMap((statusMap.status, statusMap.requests)) FROM sum_map;
```

Comment:
Tuple of Arrays looks less natural than Array of Tuples.
But if we support both it can be Ok.

Reply:
Not sure what you meant by the comment. Tuple of arrays is required in this case because SimpleAggregateFunction needs it.

Reply:
But why SimpleAggregateFunction cannot work on Array Of Tuples?

Reply:
It checks return type of sumMap.

Reply:
I think I should rephase this, as I understand in SimpleAggregateFunction input an output types should be same, and the output of sumMap is a tuple.